pipelines/tekton/pipeplines_provider.go

```diff
@@ -11,7 +11,7 @@ import (
 	"github.com/tektoncd/cli/pkg/taskrun"
 	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
 	corev1 "k8s.io/api/core/v1"
-	"k8s.io/apimachinery/pkg/api/errors"
+	errors "k8s.io/apimachinery/pkg/api/errors"
```

Comment:
Is this necessary? I don't see where you are using it in this file.

Reply:
Right, leftover from another WIP change.